README.md

```diff
@@ -159,17 +159,24 @@ Navigate your browser to [`http://localhost:5988/medic/login`](http://localhost:
 
 ### Testing locally with devices
 
-Follow the steps below to use an Android device with a development build of your application. This process is relevant when running v3.5.0 or greater of the Core Framework since it relies on service workers, which requires a valid HTTPS certificate. These steps will make your developer build accessible from your Android device by giving it a trusted URL created by _ngrok_.
+Follow the steps below to use an Android device with a development build of your application. This process is relevant when running v3.5.0 or greater of the Core Framework since it relies on service workers, which requires a valid HTTPS certificate. Use either `serveo` or `ngrok` to make your developer build accessible from your Android device by giving it a trusted URL.
 
-1. Create a ngrok account at https://ngrok.com/
-1. Follow instructions on downloading and linking your computer to your ngrok account.
-1. Start the webapp. This can be via docker, grunt, debug, horti, etc....
-1. Run ngrok and forward it towards the port you are running the webapp on.
-    * EX: For running webapp in docker locally using the docker instructions above `$ ./ngrok http 443`. This will forward the traffic from your ngrok url on https to 443 on your local machine. </br>
-    * EX: For running via horti, or grunt where the api starts on port 5988. `$ ./ngrok http 5988` This will forward the traffic from your ngrok url on https to 5988 on your local machine.
-    * Example output from ngrok: Forwarding https://1661304e.ngrok.io -> http://localhost:5988
-1. You can then enter the ngrok generated url(https://1661304e.ngrok.io) into our [android app](https://github.com/medic/medic-android) or browser and connect to your local dev environment.
+1. Start the api. This can be via docker, grunt, debug, horti, etc.
+2. Start up serveo or ngrok (instructions below)
+3. This will output a generated URL which you can enter into our [android app](https://github.com/medic/medic-android) or browser and connect to your local dev environment.
+
+#### serveo
+
+* To connect to an API running via `grunt` or `horti`, execute `ssh -R 443:localhost:5988 serveo.net`
+* To connect to an API running via `Docker`, execute `ssh -R 443:localhost:443 serveo.net`
```

Comment:
Using `ssh -R 443:localhost:443 serveo.net` this causes a 400 with an error shown below. That only seems to occur when I have cleared cache and load the site for the first time.  Using `ssh -R 80:localhost:5988 serveo.net` works for docker environments too without the above error. 
 


![image](https://user-images.githubusercontent.com/10531066/62705630-bea4b900-b9bb-11e9-93fc-1b4df43b0d9a.png)

Reply:
Do you run Docker at 5988? Isn't the default 80 or 443? Maybe it should be just `80:localhost:80` because by the time the request gets to us it's no longer HTTPS...

Reply:
I read into more about how docker networking works and maybe I understand why this is happening now. Because my container is running on the host network the ports are exposed as if I was just running from node.  That's why it needs to be 5988 for me. This probably changes based on OS since mac is different. I wonder if @Hareet  could give us a clearer answer.

Reply:
@garethbowen  I'm running the new compose file that @Hareet  has shared and now I'm unable to get serveo to work with any sort of port combination. 

`ssh -R newt:80:localhost:5988 serveo.net` does not work the port is not open with the latest compose. 

` ssh -R newt:443:localhost:443 serveo.net` does not work and responds with the bad request above. 

` ssh -R newt:443:localhost:80 serveo.net` endlessly redirects.

`ssh -R newt:80:localhost:443 serveo.net` shows the error above. 

I can get it to work by adding 5988 to the compose.yml and then relaunching compose and running serveo.

Reply:
@newtewt 5988 is not exposed anymore. Sorry, had not seen there was active use for it. API is not directly exposed in prod.

I'll ping you on Slack to get a brief summary of what we are trying to do here to make sure production will be fine